RELEASE.md

```diff
@@ -1,5 +1,14 @@
 # Releases #
 
+## Release 1.1.3 (2015-09-??) ##
+1. Updated the version string on the Wadl2Checker and Wadl2Dot CLI tools.
+1. Added the new `xsd-grammar-transform` (`g`) option to the Wadl2Checker CLI tool in order to transform the XML after validation.
+1. Added support for the `rax:metadata` extension.
+The `rax:metadata` extension can be configured on a WADL as an attribute at the resource level.
+Due to a bug in Xerces that prevents multiple Schema elements with the same `targetNamespace` from operating correctly,
+the `http://docs.rackspace.com/metadata/api` namespace should be considered reserved.
+It can **NOT** be used elsewhere if this extension is enabled.
```

Comment:
I think simply stating what the metadata extension is is good enough.  This isn't really a place where we list bugs.